docker-compose.base.yml

```diff
@@ -283,7 +283,7 @@ services:
     temporal:
         restart: on-failure
         environment:
-            - DB=postgresql
+            - DB=postgres12_pgx
```

Comment:
Why this one though, seems like the database is the same?

Reply:
I think the rest of the changes look very good :+1:

Reply:
It's because Temporal changed the driver or its name – with `postgresql` it will crash on startup. I'll check what's the latest one available.

Reply:
Just checked – there's `postgres12` and `postgres12_pgx`, I'll leave pgx, as it seems to be working well.